index.html

```diff
@@ -11,12 +11,26 @@
   <body>
     <header>
       <h1 id="logo">Aziz Mejri</h1>
-      <button type="button" class="menu-bars">
+      <button onclick="showMobileMenu();" type="button" class="menu-bars">
         <i class="fas fa-bars"></i>
       </button>
       <nav id="desktop-nav">
         <ul>
-          <li><a class="nav-link" href="#projects">Portoflio</a></li>
+          <li><a class="nav-link" href="#projects">Portfolio</a></li>
+          <li><a class="nav-link" href="#about-me">About</a></li>
+          <li>
+            <a class="nav-link" href="#contact"
+              >Contact <i class="far fa-envelope"></i
+            ></a>
+          </li>
+        </ul>
+      </nav>
+      <nav id="mobile-nav" class="hide">
+        <button onclick="hideMobileMenu();" type="button" class="close">
+          <i class="fas fa-lg fa-times"></i>
```

Comment:
Please try to strictly stick to the original template! The close icon is not the same as in  [Figma](https://www.figma.com/file/l7SqJ3ZfkAKih9sFxvWSR4/Microverse-Student-Project-1?node-id=1%3A1471) template
## Original:
![image](https://user-images.githubusercontent.com/77038610/142716844-0f9a4849-e054-44a1-8087-a7a798baa74c.png)
## Yours:
![image](https://user-images.githubusercontent.com/77038610/142716837-5fa0f408-65d8-4006-9825-3f125d52f0cd.png)
  
- [ ] I kindly suggest you to change the style of the close menu button!